### ISSUE TITLE
How to integrate with Facebook Ads

### ISSUE THREAD

**Issue by vanhung1710** [AppLovin/AppLovin-MAX-SDK-Android#49]
I investigated the Android MAX samples but I have not found facebook_adapter library document yet. As I see, we need this lib to integrate Audience Network.

**Comment by thomasmso** [comment]
Hi. Please check out our docs [here](https://dash.applovin.com/documentation/mediation/android/mediation-adapters?network=FACEBOOK_MEDIATE). Thanks!

**Comment by vanhung1710** [comment]
I follow this guide:
https://dash.applovin.com/documentation/mediation/android/mediation-adapters?network=OGURY_PRESAGE_NETWORK
But it doesn't work for me.

**Comment by vanhung1710** [comment]
I used applovin-sdk 10.3.1 and facebook-adapter 6.6.0.1

**Comment by thomasmso** [comment]
What doesn't work for you?

The correct URL is https://dash.applovin.com/documentation/mediation/android/mediation-adapters?network=FACEBOOK_MEDIATE, btw

**Comment by vanhung1710** [comment]
Right. But when I use SDK 10.3.3, it works fine. Thanks for your support.
